packages/components/src/internal/components/domainproperties/assay/actions.ts

```diff
@@ -27,11 +27,11 @@ import { isAssayEnabled } from '../../../app/utils';
 
 import { AssayProtocolModel } from './models';
 
-export function saveAssayDesign(model: AssayProtocolModel): Promise<AssayProtocolModel> {
+export function saveAssayDesign(model: AssayProtocolModel, containerPath?: string): Promise<AssayProtocolModel> {
```

Comment:
minor: from the change change in the last file, it looks like it is always passing through a contianerPath, so should this param be made required? and then can remove the `?? model.container` part from line 34 below.

Reply:
Done and removed from export out of ui-components.  We'll see if anything complains.